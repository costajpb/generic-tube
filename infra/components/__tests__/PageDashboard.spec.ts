import snapshowWrapper from "@/infra/__tests__/snapshotWrapper"
import PageDashboardVue from "../PageDashboard.vue"

describe('infra/components/PageDashboard', () => {
    it('should render', () => {
        expect(snapshowWrapper(PageDashboardVue, {}, {
            categories: Promise.resolve({})
        })).toMatchSnapshot()
    })
})