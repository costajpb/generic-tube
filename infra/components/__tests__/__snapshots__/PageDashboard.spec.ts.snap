// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`infra/components/PageDashboard > should render 1`] = `
<div>
  <div
    class="_categories_8ae98e"
  >
    
    <div>
      <div
        class="_wrapper_587f33 _name_9b2e19"
      >
        
        
      </div>
      <ol
        class="_list_9b2e19"
      >
        
        <div
          class="_wrapper_587f33 _show_9b2e19"
        >
          
          
        </div>
        <div
          class="_wrapper_587f33 _show_9b2e19"
        >
          
          
        </div>
        <div
          class="_wrapper_587f33 _show_9b2e19"
        >
          
          
        </div>
        <div
          class="_wrapper_587f33 _show_9b2e19"
        >
          
          
        </div>
        <div
          class="_wrapper_587f33 _show_9b2e19"
        >
          
          
        </div>
        
      </ol>
    </div>
    <div>
      <div
        class="_wrapper_587f33 _name_9b2e19"
      >
        
        
      </div>
      <ol
        class="_list_9b2e19"
      >
        
        <div
          class="_wrapper_587f33 _show_9b2e19"
        >
          
          
        </div>
        <div
          class="_wrapper_587f33 _show_9b2e19"
        >
          
          
        </div>
        <div
          class="_wrapper_587f33 _show_9b2e19"
        >
          
          
        </div>
        <div
          class="_wrapper_587f33 _show_9b2e19"
        >
          
          
        </div>
        <div
          class="_wrapper_587f33 _show_9b2e19"
        >
          
          
        </div>
        
      </ol>
    </div>
    <div>
      <div
        class="_wrapper_587f33 _name_9b2e19"
      >
        
        
      </div>
      <ol
        class="_list_9b2e19"
      >
        
        <div
          class="_wrapper_587f33 _show_9b2e19"
        >
          
          
        </div>
        <div
          class="_wrapper_587f33 _show_9b2e19"
        >
          
          
        </div>
        <div
          class="_wrapper_587f33 _show_9b2e19"
        >
          
          
        </div>
        <div
          class="_wrapper_587f33 _show_9b2e19"
        >
          
          
        </div>
        <div
          class="_wrapper_587f33 _show_9b2e19"
        >
          
          
        </div>
        
      </ol>
    </div>
    <div>
      <div
        class="_wrapper_587f33 _name_9b2e19"
      >
        
        
      </div>
      <ol
        class="_list_9b2e19"
      >
        
        <div
          class="_wrapper_587f33 _show_9b2e19"
        >
          
          
        </div>
        <div
          class="_wrapper_587f33 _show_9b2e19"
        >
          
          
        </div>
        <div
          class="_wrapper_587f33 _show_9b2e19"
        >
          
          
        </div>
        <div
          class="_wrapper_587f33 _show_9b2e19"
        >
          
          
        </div>
        <div
          class="_wrapper_587f33 _show_9b2e19"
        >
          
          
        </div>
        
      </ol>
    </div>
    
  </div>
  <!--v-if-->
</div>
`;
